rct229/rule_engine/rule_list_base.py

```diff
@@ -130,10 +130,9 @@ def list_filter(self, context_item, data):
 
         Returns
         -------
-        list of UserBaselineProposedVals
-            A filtered list of context trios
+        bool True iff context_item should passed through the filter.
```

Comment:
change `iff` to `if`

Reply:
The `iff` was intentional but probably a bad idea. In mathematical-speak, if means "if and only if".  I will write it out.